### PR TITLE
Added ArtSocks demo

### PR DESCRIPTION
**Pull request by dziemid**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Please raise a [Pull Request](https://github.com/wizztjh/awesome-webvr/pulls) to
 
 ### Demos
 
+- [ArtSocks](https://art-socks.herokuapp.com/demo.html) - An experiment with VR and web sockets by [dziemid](https://twitter.com/dziemid) (Google Cardboard)
```
Comment (tjiahen) [inline readme.md]: can you add in more details on what it does?

Comment (dziemid) [inline readme.md]: sure - updated README with more details - let me know if that is what you are looking for.

In more detail: it is proof of concept for virtual gallery using ThreeJS in VR - "features" -  display art work, changing between art works. Using virtual pointer to share where the user is looking and web sockets to sync multiple clients.

Comment (tjiahen) [inline readme.md]: "An experimental VR art gallery with virtual pointer to share where the user is looking using web sockets by dziemid (Google Cardboard)" Sounds good for you?

Comment (dziemid) [inline readme.md]: Perfect :-)

Comment (tjiahen) [inline readme.md]: :+1: Just tested it out with my oculus rift. Very interesting. You are using webvr boilerplate?

Comment (dziemid) [inline readme.md]: Exactly, webvr boilerplate. Good to hear it works on oculus, I've only had a chance to test on cardboard :-) - For context - here is short blog post about the demo gregdmd.com/blog/2015/11/24/campjs-project-artsocks-vr

**Pull request merged**